tide_core.module

```diff
@@ -189,3 +189,16 @@ function tide_core_entity_bundle_create($entity_type_id, $bundle) {
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function vicgovau_core_form_content_moderation_entity_moderation_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  // @todo It should implement libraries-override key under a custom theme, but we don't have custom theme.
```

Comment:
Can you explain this @vincent-gao? if it's not something we can ever do because we don't have a custom theme then remove the todo.

Reply:
@anthony-malkoun sorry. no further plan for this currently. deleted.